GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -391,6 +391,10 @@ public void unregisterView(View view) {
         mRenderableViewGroup.removeView(view);
     }
 
+    public GVRContext getGVRContext() {
```

Comment:
You can see why getGVRContext() is needed in this file: https://github.com/danke-sra/GearVRf-Demos/blob/script-dev/gvr-lua/src/org/gearvrf/sample/gvrlua/GearVRLua.java

I agree that GVRContext should be made available regardless of hierarchy. However, it was assumed that it is not needed in sub-class of GVRActivity (which owns GVRContext). This just corrects this issue.